CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-### Unreleased
+### 2.1.0 2017-07-25
```

Comment:
Why is the version being updated in a feature/fix branch? The version should be getting updated as part of the prep and tag for release.